dataland-frontend/src/frameworks/FrontendFrameworkRegistryImports.ts

```diff
@@ -0,0 +1,7 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
```

Comment:
Was this the deleted FrameworkRegistryImport.ts file?

Reply:
When importing the framework registry, you also import all the TS files for the view config. This includes some ".vue" files. This leads to e.g., cypress crashing because it does not know what to do with ".vue" files. Because of this, there are 2 framework registries. One also imports ".vue" files for the frontend, the other does not (so it can be used in e2e-tests).

Reply:
Ah ok. And it was introduced here, because GDV is our first framework with the toolbox and the problem was probably discovered here, because this is the first framework.